test/e2e/utils.py

```diff
@@ -0,0 +1,36 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+from deadline.job_attachments._aws.deadline import get_queue
+from deadline.job_attachments import download
+from deadline_test_fixtures import (
+    DeadlineClient,
+    Job,
+)
+
+from e2e.conftest import DeadlineResources
+
+
+def get_job_output(
```

Comment:
Can we add some function documentation, and also change the name? Right now it is not clear this function WAITS for the job to be completed!